docs/build.md

```diff
@@ -0,0 +1,51 @@
+# Building a go binary that includes golang-fips patches
+
+## Env variables
+
+`GOEXPERIMENT=strictfipsruntime` helps ensure a fips compliant binary is build.  Note that this is functionally equivalent of
+
+```
+GO_BUILDTAGS+="goexperiment.strictfipsruntime"
+```
+
+You can export more Go flags like `GO_GCFLAGS="-N -l"` to add symbols in the final binary.
+
+## Native build - Method 1
+
+The easiest way to do is to simply clone this repo and run make. see a more comprehensive script at `scripts/build1.sh`
+
+```
+git clone https://github.com/golang-fips/go.git golang-fips
+cd golang-fips
+./scripts/full-initialize-repo.sh go$GOLANG_VER
+cd go/src
+./make.bash --no-clean
+```
+
+## Native build - Method 2
+
+Another way is to directly apply fips patches on  downloaded go and golang-fips binaries.  see full script at `scripts/build2.sh`
```

Comment:
Formatting: there's an extra space after 'patches on' and then capitalize any word after punctuation.

Reply:
Fixed